frontend/src/components/ApplicationForm.jsx

```diff
@@ -1,10 +1,53 @@
 import Form from "react-bootstrap/Form"
+import Container from 'react-bootstrap/Container'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
+import 'bootstrap/dist/css/bootstrap.min.css';
+import React, { useState } from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
 
 const ApplicationForm = (props) => {
   // initialize state below this line
 
+  // keeps track of which checkboxes are clicked
+  const [roles, setRoles] = useState(
+    {
+    TestDev: false,
+    TestDes: false,
+    Dev: false,
+    Des: false,
+    ProdMan: false
+    }
+  );
+  
+
   // create any event handler functions below this line
 
+  let handleCheckbox = (label) => {
+
+    /* takes in which checkbox called this function and flips
+     the boolean in roles*/
+    switch(label) {
```

Comment:
rather than using a switch-case, you can have label be the value that is passed in.

For example:
for Test Developer, you will pass in "TestDev" and just directly access the field in the state.

Quick tip: maybe have an id for each form group. For the TestDeveloper form group have the id be "testDev." When you pass in the event, to this handler you can just refer to the id using the line... e.target.id

Reply:
That's what I initially had my code as, but I changed it when it wasn't working. I think i forgot to actually pass in the event.